app/screens/CustomDrawerContent.tsx

```diff
@@ -54,41 +55,45 @@ import HTML from 'react-native-render-html';
 import { ThemeContext } from 'styled-components/native';
 import { useAppSelector } from '../../App';
 import { isFutureDate } from '../services/childCRUD';
-import { formatDate,addSpaceToHtml } from '../services/Utils';
+import { formatDate, addSpaceToHtml } from '../services/Utils';
 
 const styles = StyleSheet.create({
   containerView: {
-    backgroundColor:secondaryColor,
-    flex:1
+    backgroundColor: secondaryColor,
+    flex: 1
   },
-  headingFlexShrink: { 
+  headingFlexShrink: {
     flexShrink: 1
   },
-  iconStyle: { 
+  iconStyle: {
     alignSelf: 'center',
     flex: 1,
-    textAlign: 'right' 
+    textAlign: 'right'
+  },
+  padding2:{
+    paddingLeft:2
   },
   scrollViewStyle: {
-    backgroundColor:bgcolorWhite2,
-    flex:1
+    backgroundColor: bgcolorWhite2,
+    flex: 1
   },
-  textStyle: { 
+  textStyle: {
     fontSize: 12,
     fontWeight: 'normal'
   },
-  touchableLeft: { 
+  touchableLeft: {
     marginLeft: 2,
-    padding: 8 
+    padding: 8
   },
-  touchableRight: { 
+  touchableRight: {
     marginRight: 2,
-    padding: 8 
+    padding: 8
   }
 })
-const CustomDrawerContent = ({ navigation }: any):any => {
+const CustomDrawerContent = ({ navigation }: any): any => {
   const { t } = useTranslation();
   const [accordvalue, onChangeaccordvalue] = React.useState(false);
+  const [aboutaccordvalue, onChangeAboutAccordvalue] = React.useState(false);
```

Comment:
Should we follow camel case naming convention? 
aboutaccordvalue  > aboutAccordValue
onChangeAboutAccordvalue > onChangeAboutAccordValue

Reply:
done